ingest/rules/curate.smk

```diff
@@ -125,6 +128,8 @@ rule subset_metadata:
         metadata_fields=",".join(config["curate"]["metadata_columns"]),
     shell:
         r"""
+        exec &> >(tee {log:q})
```

Comment:
Missing a `log` directive

Reply:
Fixed in https://github.com/nextstrain/mumps/pull/31/commits/63bf0f2ba08d221377d9e85304f10fffcfaef6f7